pkg/kubectl/cmd/get.go

```diff
@@ -69,6 +72,7 @@ func NewCmdGet(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 			err := RunGet(f, out, cmd, args)
 			cmdutil.CheckErr(err)
 		},
+		ValidArgs: validArgs,
```

Comment:
There was definitely thought, in the back of my mind, of making cobra enforce validity one day...

These completions will suggest `kubectl get pods $NAME`, not `kubectl get pods/$NAME`.  Do you think I should switch from a space separator to a / separator in my completions.

There is obviously no way to know, at compile time, all of the resource/name options...   So maybe the potential someday cobra validOptions validator would have to be a prefix validator, or an over writable validator function...